Doc/library/queue.rst

```diff
@@ -57,8 +57,8 @@ The :mod:`queue` module defines the following classes and exceptions:
    *maxsize* is less than or equal to zero, the queue size is infinite.
 
    The lowest valued entries are retrieved first (the lowest valued entry is the
-   one returned by ``sorted(list(entries))[0]``).  A typical pattern for entries
```

Comment:
The two spaces after the period here are deliberate. Not that elsewhere in this file, periods have two spaces after them. This is used by some editors (including emacs) when a paragraph is re-flowed. I suggest putting this "extra" space back.

PEP 8 mentions this for comments. https://devguide.python.org/documentation/style-guide/ says one or two spaces is acceptable, but in this case I think we should match the rest of the file.

Reply:
@OTheDev : yes, go ahead and open a PR.

Reply:
I beg to differ, @ericvsmith. We do not need to encourage contributors to "fix" one or two spaces after period. There's better use of both contributor time and review time. Also, what has PEP-8 with .rst files to do? AFAICS, nothing.

Reply:
> PEP 8 mentions this for comments.

See python/peps#3024.